README.md

```diff
@@ -1 +1,26 @@
+# Dumb things!
+
 A dumb repo with dumb code.
+
+A dumb subheader!
+=============
+
+
+
+
+1. A
+2. Dumb
+3. List
+4. !
+
+## Another dumb subheader!
+
+http://foo.bar/baz
+
+Hi [foo] bar.
+
+[foo]: bar
+[foo]: qux
```

Comment:
Do not use definitions with the same identifier (22:1)<br>Move definitions to the end of the file (after the node at line `25`)